CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 ### New
 
+- **General**: Provide Prometheus metrics for total number of triggers per trigger type. ([#3663](https://github.com/kedacore/keda/issues/3663))
```

Comment:
Just a suggestion
```suggestion
- **General**: Provide Prometheus metric with indication of total number of triggers per trigger type in `ScaledJob`/`ScaledObject` ([#3663](https://github.com/kedacore/keda/issues/3663))
```

Reply:
I'll try to see if I can do that.

Reply:
[I  added 1 e2e test to Prometheus](https://github.com/kedacore/keda/blob/main/tests/internals/prometheus_metrics/prometheus_metrics_test.go) when I fixed one issue we had (in the metrics server), you could extend that test to cover operator and metrics server or just duplicate it having one e2e test per pod. The logic to parse the prometheus metrics is already done in the test

Reply:
We would need to add a service for the operator pod as well, wouldn't we?

Reply:
Added the e2e test, @JorTurFer @tomkerkhove